.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,104 @@
+name: dotnet
+
+on:
+  push:
+    paths:
+      - "samples/dotnet/**"
+      - ".github/workflows/dotnet.yml"
+  schedule:
+    - cron: '0 7 * * *'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: miracl/oidc-samples/samples/dotnet
+  DOCKER_BUILD_CONTEXT: samples/dotnet
+  SAMPLE_IMAGE: ghcr.io/miracl/oidc-samples/samples/dotnet:${{ github.sha }}
+  PROXY_IMAGE: ghcr.io/miracl/oidc-samples/proxy:latest
+  TESTS_IMAGE: ghcr.io/miracl/oidc-samples/integration-tests:latest-windows
+  CLIENT_ID: ${{ secrets.CLIENT_ID_PK }}
+  CLIENT_SECRET: ${{ secrets.CLIENT_SECRET_PK }}
+  CID_FILE: sample.cid
+
+jobs:
+  build-dotnet:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup MSBuild
+        uses: microsoft/setup-msbuild@v1
+      - name: Setup NuGet
+        uses: NuGet/setup-nuget@v1.0.5
+      - name: Setup VSTest
+        uses: darenm/Setup-VSTest@v1
+      - name: Restore Packages
+        run: nuget restore samples/dotnet/OidcSample.sln
+      - name: Build Solution
+        run: |
+          msbuild.exe samples/dotnet/OidcSample.sln /p:platform="Any CPU" /p:configuration="Release"
+
+  build-docker:
+    needs: build-dotnet
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Log in to GitHub Docker Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          tags: |
+            type=raw,value=${{ github.sha }}
+      - name: Docker login, build & push
```

Comment:
Can't we use docker/build-push-action@v2?

Reply:
no.. it gives "error: Error response from daemon: Windows does not support privileged mode" (https://github.com/miracl/oidc-samples/runs/4315246435?check_suite_focus=true). It doesn't show to run it with it but... The only way to make it to work on Windows is this one